include/zephyr/net/wifi_mgmt.h

```diff
@@ -402,6 +402,14 @@ struct wifi_connect_req_params {
 	uint8_t bssid[WIFI_MAC_ADDR_LEN];
 	/** Connect timeout in seconds, SYS_FOREVER_MS for no timeout */
 	int timeout;
+	/** anonymous identity */
+	const uint8_t *aid;
```

Comment:
`anon_id` is better to disambiguate with `Association ID`

Reply:
I can change it